fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -70,6 +70,8 @@
     <string name="preferences_debug_settings_shopping_experience" translatable="false">Enable Shopping Experience</string>
     <!-- Label for enabling translations -->
     <string name="preferences_debug_settings_translations" translatable="false">Enable Firefox Translations</string>
+    <!-- Label for enabling Firefox Suggest search suggestions -->
```

Comment:
```suggestion
    <!-- Label for enabling Firefox Suggest -->
```

(It's more accurate to say that it enables the entire feature—ingestion and the new settings, not just search suggestions).